style.css

```diff
@@ -3282,4 +3290,4 @@ article.panel-placeholder {
 	.twentyseventeen-front-page .entry-content blockquote.alignright {
 		margin-right: -20%;
 	}
-}
\ No newline at end of file
+}
```

Comment:
This is probably coming from your editor, but can you please re-add the newline at the bottom of the file?